manifoldco/manifold-subscription
# add ui error states

## Change

- added various UI error states

TODO:
- [x] probably some cleanup 
- [x] there's probably some not good states in here somewhere
- [ ] `gateway.post` needs to return errors because right now it swallows them and I can't show a UI error



## Testing

- make some invalid gql queries
- remove some props that are required
- go offline after the page has loaded but before subscribing

Does the UI error in way you would expect?
